tauri/Cargo.toml

```diff
@@ -45,6 +49,7 @@ futures = "0.3.28"
 regex = "1.8.1"
 anyhow = "1.0.71"
 secrets = "1.2.0"
+reqwest = { version = "0.11", features = ["json"] }
```

Comment:
can you add the `tokio-rustls` feature here?
I think that should get rid of the extra dependency that showed up on Cargo.lock, and it's also more consistent to avoid using two different TLS implementations

Reply:
Im not seeing that feature here: https://docs.rs/reqwest/latest/reqwest/
Where can i find more about that?